### ISSUE TITLE
Authentication Not Working with 1.5.3

### ISSUE THREAD

**Issue by naikmithilesh** [influxdata/influxdb#9942]
I have installed InfluxDB lately on Ubuntu 16.04.

After un-commenting and setting auth-enabled = true authentication should be required even by CLI. However CLI is allowing to query all the data without getting credentials.

I have gone through other reports of uncommenting  [http] line. I assume those tickets were older since the default config file shipped with the version I installed does not have it commented by default.
Refer to screenshot below.

![image](https://user-images.githubusercontent.com/39955217/40976995-573894f0-68ed-11e8-97df-997cc6ed8f50.png)

I have restarted service and still unable to have authentication enabled. below is the screenshot for CLI:
![image](https://user-images.githubusercontent.com/39955217/40977169-d12592e0-68ed-11e8-892c-9f3daaa0dee7.png)

This is used to work with the older version of influxDB 0.10.0. But since I have upgraded to 1.5.3, authentication has stopped. Any help is appreciated.

**Comment by e-dard** [comment]
@naikmithilesh Hi, I'm not able to reproduce this. A couple of possibilities:

 1) for some reason the server hasn't been restarted properly;
 2) The server has been started with a different config file;
 3) The server has been started with the environment variable `INFLUXDB_HTTP_AUTH_ENABLED = false` in scope.

Can you double check all those things?

**Comment by CAFxX** [comment]
After uncommenting `auth_enabled`, did you create one or more admin users? As the [docs](https://docs.influxdata.com/influxdb/v1.5/query_language/authentication_and_authorization/#set-up-authentication) say, that's required for authorization to start working:

> InfluxDB will enforce authentication once there is an admin user.

**Comment by naikmithilesh** [comment]
@CAFxX Yes I have Admin users already created - I will still create one more just to be sure again.

@e-dard 

1. This is not happening at one particular moment. I say this because I have restarted both server and machine multiple times. Also, I have tried restarting it through service and through command prompt as well.

2. I have created a copy of a config file and updated the copy to have Auth enabled. A point to note here is that I have updated the environment variable to point to the custom config file. However there is no discrepancy in the file which is picked. I have ensured this by updating other fields in same file and that reflects in the DB Configs. This confirms that env variable is point to correct path hence the correct config is picked up.

3. Yes, good point. I missed this thing to check. However, I have not manually created/updated any such environment variable and hence I assume this should not the reason. Still, I will check this once more and manually set it to true in variables just to be sure.
